src/pages/MonthlyGood/MonthlyGoodLandingPage.vue

```diff
@@ -314,14 +321,47 @@ export default {
 // Experiment Styles - GROW-11
 .mg-hero.experiment {
 	::v-deep .overlay-content {
+		top: 2rem;
+		bottom: auto;
+		transform: none;
+		@include breakpoint(large) {
+			margin-left: 1rem;
+		}
+
+		@include breakpoint(medium only) {
+			top: 17rem;
+		}
+
+		@include breakpoint(small only) {
+			top: 17rem;
+		}
+
 		.overlay-column {
 			background-color: white;
-			@include breakpoint(large) {
-				max-width: 28rem !important;
+			padding: 2rem 2rem 1.25rem;
+			border-radius: 1rem;
+			max-width: 26rem !important;
+
+			@include breakpoint(large only) {
+				max-width: 22rem !important;
+			}
+
+			@include breakpoint(medium only) {
+				margin: 0 auto;
+			}
+
+			@include breakpoint(small only) {
+				margin: 0 auto;
 			}
 		}
 	}
 
+	::v-deep .images > div {
+		height: 32rem;
```

Comment:
Which div is this? It's probably better to let the height of the div come from the size of the elements within it, rather than using a fixed size.